frontend/src/metabase/ui/components/icons/Icon/Icon.tsx

```diff
@@ -39,7 +40,17 @@ export const Icon = forwardRef<SVGSVGElement, IconProps>(function Icon(
     />
   );
 
-  return tooltip ? <Tooltip label={tooltip}>{icon}</Tooltip> : icon;
+  return tooltip ? (
+    <Tooltip
+      style={{ whiteSpace: multiline ? "unset" : "nowrap" }}
```

Comment:
At least to my way of thinking, this line seems to support a non-multiline code path - but if I understand correctly, there isn’t one, and doesn’t need to be one. That might confuse someone, so I’d think about either going all-in with multiline (don’t even make it a variable) or making it a prop with a default value. 

This isn’t a blocker, though.

Reply:
you're right, this'll be cleaner